test/test_array_function.py

```diff
@@ -293,6 +301,11 @@ def test_numpy_negative(func_interface):
     assert func_interface.negative(X-Y-1) == 1-X+Y
     assert numpy.all(func_interface.negative(poly) == [[-X, Y], [4, -Y]])
 
+def test_numpy_nonzero(func_interface):
```

Comment:
The difference between `func_interface` and `interface` is that the former will test `numpy.nonzero(poly)` and `numpoly.nonzero(poly)`, while the latter will also attempt `poly.nonzero` (or `poly.__nonzero__` if that doesn't exist.)
As `numpoly` inherits `numpy` attributes, and `<ndarry>.nonzero` is a thing, this test should be:
```suggestion
def test_numpy_nonzero(interface):
```